pkg/config/config_test.go

```diff
@@ -617,6 +620,41 @@ func (s *MySuite) TestImportBlueprint(c *C) {
 		Equals, expectedSimpleBlueprint.DeploymentGroups[0].Modules[0].ID)
 }
 
+func (s *MySuite) TestImportBlueprint_LabelValidation(c *C) {
```

Comment:
Must have missed this earlier. I don't actually see a test for success here. Please add one. Also consider a test for edge cases like empty string or a capital "N~".

Approving on assumption that you address this concern.

Reply:
Good idea - done.